### PR TITLE
Add workflow package basic docs

### PR DESCRIPTION
**Pull request by andresgomezfrr**

### CODE REVIEW

File: pages/wdp/1.0.X/workflow_package.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+You can check [Kafka Topic Spec](./wdp_1.0.X_batuta_kafka.html)
+
+
+### Druid
+
+On this folder, you must create a file per each streaming indexer, using the data source name as the filename.
+
+*Format: `${datasource_name}.json`*
+
+The content of the file is a [Streaming Indexer spec](./wdp_1.0.X_indexer_index.html#stream-indexing). It is defining what and how the data is indexed.
```
Comment (cjmateos) [inline pages/wdp/1.0.X/workflow_package.md:60]: `It is defining what and how the data is indexed.` -> `It defines the data to index and how it is indexed.`

File: pages/wdp/1.0.X/workflow_package.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+### Streams
+
+This folder has the three main stream-plans (normalizer, enricher, zz-cep). Inside these files you define the stream-plan configuration to each service, using their definitions:
```
Comment (cjmateos) [inline pages/wdp/1.0.X/workflow_package.md]: `Inside these files you define the stream-plan configuration to each service`-> `On these files, you define the stream-plan configuration for each service basing of it definitions`

File: pages/wdp/1.0.X/wizz-in/workflow_package.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+title: Workflow Package Management
+permalink: /wdp_1.0.X_workflow_package_management.html
+toc: true
+---
+
+## Overview
+
+This section show the [Workflow Package](./wdp_1.0.X_workflow_package.html) Management. Allowing to download, upload and delete the Workflow Packages by organization.
```
Comment (cjmateos) [inline pages/wdp/1.0.X/wizz-in/workflow_package.md]:
```suggestion
This section shows the [Workflow Package](./wdp_1.0.X_workflow_package.html) Management. It allows to download, upload and delete the Workflow Packages by organization.
```

File: pages/wdp/1.0.X/wizz-in/workflow_package.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+At bottom of organization main page, we can see this:
```
Comment (cjmateos) [inline pages/wdp/1.0.X/wizz-in/workflow_package.md]:
```suggestion
At the bottom of the organization main page, we can see this section:
```

File: pages/wdp/1.0.X/wizz-in/workflow_package.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+## Operations
+
+Using the interface, you can do the different operations:
+
+### UPLOAD
+
+Using the `+`, you can select a [workflow_package.tar.gz](./wdp_1.0.X_workflow_package.html) and upload to the platform.
+
+{% include note.html content="The Workflow Package add/update the different WDP stack resources to this organization. Doesn't remove!" %}
```
Comment (cjmateos) [inline pages/wdp/1.0.X/wizz-in/workflow_package.md]:
```suggestion
{% include note.html content="The Workflow Package adds/updates WDP stack resources of the organization where it is uploaded. This action doesn't remove resources created before." %}
```

File: pages/wdp/1.0.X/wizz-in/workflow_package.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+The `UPLOAD` return a message where you can see which stack resource are created/updated or if something have errors.
```
Comment (cjmateos) [inline pages/wdp/1.0.X/wizz-in/workflow_package.md]:
```suggestion
After `UPLOAD` is performed, it returns a message where you can see which stack resources are created/updated or if something went wrong.
```

File: pages/wdp/1.0.X/wizz-in/workflow_package.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+{% include image.html file="./wizz-in/wizz-in-workflow-management-upload.png" url="./images/wizz-in/wizz-in-workflow-management.png" alt="" caption="Wizz-In Workflow upload example" %}
+
+### DOWNLOAD
+
+Generate a new workflow_package.tar.gz with the current stack resources status to this organization. and download it.
```
Comment (cjmateos) [inline pages/wdp/1.0.X/wizz-in/workflow_package.md]:
```suggestion
Creates a new workflow_package.tar.gz file with the current stack resources status of the related organization and downloads it.
```

File: pages/wdp/1.0.X/wizz-in/workflow_package.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+### DELETE
+
+Remove all current stack resources to this organization.
```
Comment (cjmateos) [inline pages/wdp/1.0.X/wizz-in/workflow_package.md]:
```suggestion
Removes every current stack resource of this organization.
```

**Review by pablocasares — APPROVED** [review]

**Review by cjmateos — APPROVED** [review]

**Pull request merged**